perun-scim/src/main/java/cz/metacentrum/perun/scim/api/entities/GroupSCIM.java

```diff
@@ -13,7 +13,10 @@
  */
 
 @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
-public class GroupSCIM extends Resource {
+public class GroupSCIM {
```

Comment:
When I look at it now, it seems, that both ``GroupSCIM`` and ``UserSCIM`` are DTO objects. So regarding our original request "we should use composition and not extension", we must make sure:
- we align also ``UserSCIM`` to use composition and not extension.
- we must prevent ``getResource()`` and ``setResource(Resource)`` to be (de)serialized to JSON, since former implementation has no property "resource" but instead its internal properties (id,externalId,meta) as a part of to level object (group/user).